tests/golden/gcp/openshift4-nodes/openshift4-nodes/machineset-infra-a.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  annotations: {}
+  labels:
+    machine.openshift.io/cluster-api-cluster: none
```

Comment:
The defaults hold two parameters with the same value (`projectName`, `infrastructureID`). To have proper testing, please set them explicitly and to distinct values.